packages/components/src/AboutDialog/AboutDialogTable.component.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import { getI18nInstance } from '../translate';
+import Skeleton from '../Skeleton';
+
+import theme from './AboutDialog.scss';
+
+const i18n = getI18nInstance();
+
+export const getColumnHeaders = () => ({
```

Comment:
This is used only once with `Object.values(getColumnHeaders())` --> can you just make it a list ?

Reply:
No because of the need to spread [here](https://github.com/Talend/ui/pull/2586/files#diff-150395fd55bae16ae173a484d3dfa842R49)